var/spack/repos/builtin/packages/apple-dtrace/package.py

```diff
@@ -0,0 +1,107 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+import os.path
+
+
+class AppleDtrace(Package):
+    """apple-dtrace: tools from Apple's version of dtrace tracing framework
+
+    Downloads Apple's version of the dtrace dynamic tracing framework
+    and installs related tools. Does not install dtrace because it
+    requires darwin kernel source headers.
+
+    Installs:
+
+    * libelf
+    * libdwarf
+    * libctf
+    * ctfconvert
+    * ctfdump
+    * ctfmerge
+    * additional headers used by memory instrumenting tool frameworks
```

Comment:
If you look at the output of `spack info` you'll see that this is probably not formatted correctly.